rocky/rocky/scheduler.py

```diff
@@ -256,5 +257,11 @@ def health(self) -> ServiceHealth:
         health_endpoint.raise_for_status()
         return ServiceHealth.model_validate_json(health_endpoint.content)
 
+    def get_task_stats(self, organization_code: str, task_type: str) -> Dict:
+        res = self.session.get(f"{self._base_uri}/tasks/stats/{task_type}-{organization_code}")
+        res.raise_for_status()
```

Comment:
You might want to throw a `SchedulerError` in case of an exception?

Reply:
This way the view can catch this exception and render an error message or an empty table with a message in case this specific request fails while the other request (task list) succeeds